concepts-workloads-controllers-deployment-pr-20170728

rootsongjc opened the pull request:
Deployment workload controller translated done!

rootsongjc committed:
concepts-workloads-controllers-deployment-pr-20170728

rootsongjc committed:
Merge remote-tracking branch 'upstream/master'

markthink commented on docs/concepts/workloads/controllers/deployment.md:
```diff
@@ -9,30 +9,50 @@ redirect_from:
+-->
+## 什么是 Deployment ？
+
+Deployment 为 [Pod]((/docs/concepts/workloads/pods/pod/)) 和 [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/)（下一代Replication Controller）提供声明式更新。
```
保留md 源格式 _Deployment_  _会生成`<u>` html 标签

markthink commented on docs/concepts/workloads/controllers/deployment.md:
```diff
@@ -9,30 +9,50 @@ redirect_from:
 * [Clean up older ReplicaSets](#clean-up-policy) that you don't need anymore
+  -->
 
+- 使用Deployment来创建ReplicaSet。ReplicaSet在后台创建pod。检查启动状态，看它是成功还是失败。
```
类似的链接未保留：(#creating-a-deployment)

rootsongjc replied:
已添加完成。

rootsongjc committed:
Merge remote-tracking branch 'upstream/master'

rootsongjc committed:
add style to words and tags

fix https://github.com/k8smeetup/kubernetes.github.io/pull/16#discussion_r130119258

rootsongjc committed:
concepts-services-networking-ingress-pr-20170730

Ingress resource translated
